docs/feature_testing.md

```
@@ -0,0 +1,399 @@
+  crossorigin="anonymous"
+></script>
+```
+
+### Verify SRI Works
+
+**Break the integrity (for testing):**
+
+1. Edit `public/packs/application-xyz789.js` (add a space)
+2. Reload page
+3. **Expected:** Browser console shows SRI error:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)